src/ProjectSystemTools/ProjectSystemTools.csproj

```diff
@@ -51,6 +45,8 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="xlf\*" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
What's the reason to create separate item groups?

Reply:
Just personal preference.